src/IssueViz.Security/HotspotsList/TableDataSource/HotspotsTableDataSource.cs

```diff
@@ -67,5 +81,55 @@ public void Dispose()
         {
             tableManager.RemoveSource(this);
         }
+
+        public void Add(IAnalysisIssueVisualization hotspot)
+        {
+            var entry = new HotspotTableEntry(hotspot);
+            tableEntries.Add(entry);
+
+            lock (sinks)
+            {
+                foreach (var sink in sinks)
+                {
+                    sink.AddEntries(new[] {entry});
+                }
+            }
+
+            var hotspotFilePaths = hotspot.GetAllLocations().Select(x => x.CurrentFilePath);
```

Comment:
GetAllLocations = new extension method in `AnalysisIssueVisualizationExtensionsTests`

Reply:
Can there be duplicate file paths in the list, or will they be de-duplicated in the event args class?

Reply:
Fixed, added ToLower + Distinct

Reply:
Use `.Distinct(StringComparer.OrdinalIgnoreCase)`

Some languages don't have a one-to-one mapping between upper and lower case characters (I think Turkish is an example) so forcing to upper/lower and comparing isn't the right solution.

Reply:
Huh, good to know 👍